src/nodes.coffee

```diff
@@ -966,6 +966,22 @@ exports.RegexLiteral = class RegexLiteral extends Literal
     val = replaceUnicodeCodePointEscapes val, {@flags}
     @value = "#{makeDelimitedLiteral val, delimiter: '/'}#{@flags}"
 
+  REGEX_REGEX: /// ^ / (.*) / \w* $ ///
+
+  astType: -> 'RegExpLiteral'
+
+  astProperties: ->
+    [, pattern] = @REGEX_REGEX.exec @value
+    return {
+      value: undefined
```

Comment:
The `undefined` for `value`/`extra.rawValue` are a little unusual, but that's what Babel generates

I looked at the `babel-parser` source code, and it seems to treat `RegExpLiteral`s like other literal types that expose a `value`/`extra.rawValue`, but for some reason it passes a `value` that's undefined (at least for all regexes I've looked at)